### PR TITLE
helm: update chart for flattening

### PR DESCRIPTION
**Pull request by wtty-fool**
Towards https://github.com/giantswarm/giantswarm/issues/9337 as part of https://github.com/giantswarm/giantswarm/issues/7897.

## Checklist
- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: helm/cert-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -39,17 +39,17 @@ spec:
...
       securityContext:
         runAsUser: {{ .Values.userID }}
         runAsGroup: {{ .Values.groupID }}
       containers:
       - name: cert-operator
-        image: "{{ .Values.Installation.V1.Registry.Domain }}/giantswarm/cert-operator:[[ .SHA ]]"
+        image: "{{ .Values.Installation.V1.Registry.Domain }}/giantswarm/cert-operator:[[ .Version ]]"
         volumeMounts:
-        - name: cert-operator-configmap
+        - name: {{ .Chart.Name }}-configmap
```
Comment (kopiczko) [inline helm/cert-operator/templates/deployment.yaml:50]: Wouldn't it be better to `{{ include "name" . }}` here?

Comment (wtty-fool) [reply helm/cert-operator/templates/deployment.yaml:50]: `"name"` is a bit nicer `.Chart.Name`, so sure.

**Comment by wtty-fool** [comment]
@rossf7 Sure! I'll create a follow-up PR for that though.

**Comment by rossf7** [comment]
@wtty-fool Great! Thanks for putting up with my cheekiness.

**Pull request merged**